cmd/generate/action/runner.go

```diff
@@ -48,53 +49,67 @@ func (r *runner) Run(cmd *cobra.Command, args []string) error {
 }
 
 func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) error {
-	data := actiontemplate.Data{
```

Comment:
The code is just executed in a loop, but otherwise stays the same. The diff does not really reflect that in all cases.